src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -95,13 +95,19 @@ public override void OnEdited()
     public override void RepositionToOrigin()
     {
         Organelles.RepositionToOrigin();
+        CalculateRotationSpeed();
     }
 
     public void CalculateRotationSpeed()
     {
         BaseRotationSpeed = MicrobeInternalCalculations.CalculateRotationSpeed(Organelles);
     }
 
+    public void RecalculatePositionAndStatistics()
```

Comment:
I commented directly on the commit here that I think this method should be removed (all use of this should go through the base Species.RepositionToOrigin method).

Reply:
MicrobeSpecies implements ICellProperties, so it has to have RecalculatePositionAndStatistics.

I could remove RecalculatePositionAndStatistics from ICellProperties and rework it so RepositionToOrigin calls CalculateRotationSpeed in every class that implements ICellProperties, so RepositionToOrigin would essentially become what RecalculatePositionAndStatistics is now.

Or, we can keep it how it is and accept that if you're interacting with a MicrobeSpecies as a cell, you call RecalculatePositionAndStatistics and if you're interacting with it as a species, you call RepositionToOrigin.

Either way works for me.

Reply:
I originally designed things to be symmetric between the cell properties and the species. So I'd prefer to just have RepositionToOrigin be the method name both in the species and the ICellProperties.